assets/xml/sequences/Sequences.xml

```diff
@@ -0,0 +1,112 @@
+<Root>
+    <Sequence Name="000_Sound_Effects" Extract="false"/>
+    <Sequence Name="001_Ambient_Effects" Extract="false"/>
+    <Sequence Name="002_Hyrule_Field_Program" Extract="false"/>
+    <Sequence Name="003_Hyrule_Field_Intro"/>
```

Comment:
These sequence names should be synced up with the names used in `sequence.h` defines. I'm not too worried about which of the two names are chosen, which ever one is the better of the two sets:
https://github.com/zeldaret/oot/blob/2e6279bc8e528833704bc6b77635f0eeae6bbe44/include/sequence.h#L6

Reply:
You mean the ones in the comments of that file?  I'm fine with either one, whichever the community is more partial to.

Reply:
should we match up to the N64 original soundtrack names or the 3D original soundtrack? From what I can tell, the N64 original soundtrack names might be fan translated, while the 3D original soundtrack seems less comprehensive.

Reply:
Can do if we make some group decisions. (Not yet done in #1509)